server/known-accounts.json

```diff
@@ -370,5 +370,141 @@
   {
     "alias": "BananoFaucet.club",
     "account": "ban_3rzg5w93ipik4aie3uzafetsfa6dnqdrrsmw9c9dussge7eb5s45xmh4o4pa"
+  },
+  {
+    "alias": "BanSlots",
+    "account": "ban_1kwin96znfqopi7be3shxcxn8qeruirob885oaya4ix5pkrnpsou4u5qbeaa"
+  },
+  {
+    "alias": "One Love",
+    "account": "ban_1heart7e8u4tnyowup9hwchx8tkfaqjiyp67si74gdanziizegf7p37jd6gf"
+  },
+  {
+    "alias": "Yag",
+    "account": "ban_1yag3k5uegjz5o3xbek83yibbb75sdne3rjtuuh3ry555pr835781eh9wecy"
+  },
+  {
+    "alias": "BeanNode",
+    "account": "ban_37ycdzmcb3wujsqkztta75wzncwbjx5qwm7io9usqx8r133c77rmtyzi4mop"
+  },
+  {
+    "alias": "FI Monke 🐒",
+    "account": "ban_1otahirsy63kwzi6e3578wm1q6d67pgcude31gkd7e31gcktuguuyhdtzton"
+  },
+  {
+    "alias": "Moonano",
+    "account": "ban_1moonanoj76om1e9gnji5mdfsopnr5ddyi6k3qtcbs8nogyjaa6p8j87sgid"
+  },
+  {
+    "alias": "Discord Beta Rewards",
+    "account": "ban_3disc5557sb9ri99h7czmn6ms5kcfsafnsxekarg1pp9f3a1ik4ndjcb9cod"
+  },
+  {
+    "alias": "Discord Halloween",
+    "account": "ban_1treatf1gc4acpgjqzg9jxthwqnew7gef5wf7mru4ffyeb9ayj61rejzs5st"
+  },
+  {
+    "alias": "Discord Santa",
+    "account": "ban_1santayq7qgtoar9s9kx9ur5jw6ustty741bxnidanf8miuakju7kqb8imwo"
+  },
+  {
+    "alias": "Dungeons and Banano",
+    "account": "ban_3dnbrpg7abg7fuhryp3jq5r56ka4aijjen6ysxushujq9r69f31ip1dbx3qc"
+  },
+  {
+    "alias": "Fantom Cold Wallet",
+    "account": "ban_3ftmco1d3sogpnq9j8gsa7t95xugtzj116hkudxj1spkw86476pzicua61ek"
+  },
+  {
+    "alias": "Fantom Hot Wallet",
```

Comment:
is Fantom an exchange? might consider to also add their wallets to `src/exchanges.json` so it gets tracked on https://bananolooker.com/exchange-tracker

Reply:
The wallet is hot wallet for wrapped Banano on Fantom. Not an exchange